{
  "cve": {
    "data_type": "CVE",
    "data_format": "MITRE",
    "data_version": "4.0",
    "CVE_data_meta": {
      "ID": "CVE-2020-4430",
      "ASSIGNER": "psirt@us.ibm.com"
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [
            {
              "lang": "en",
              "value": "CWE-22"
            }
          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "url": "https://www.ibm.com/support/pages/node/6206875",
          "name": "https://www.ibm.com/support/pages/node/6206875",
          "refsource": "CONFIRM",
          "tags": [
            "Patch",
            "Vendor Advisory"
          ]
        },
        {
          "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/180535",
          "name": "ibm-drm-cve20204430-dir-trav (180535)",
          "refsource": "XF",
          "tags": [
            "VDB Entry",
            "Vendor Advisory"
          ]
        }
      ]
    },
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "IBM Data Risk Manager 2.0.1, 2.0.2, 2.0.3, and 2.0.4 could allow a remote authenticated attacker to traverse directories on the system. An attacker could send a specially-crafted URL request to download arbitrary files from the system. IBM X-Force ID: 180535."
        }
      ]
    }
  },
  "configurations": {
    "nodes": [
      {
        "operator": "OR",
        "cpe_match": [
          {
            "vulnerable": true,
            "cpe23Uri": "cpe:2.3:a:ibm:data_risk_manager:2.0.2:*:*:*:*:*:*:*"
          },
          {
            "vulnerable": true,
            "cpe23Uri": "cpe:2.3:a:ibm:data_risk_manager:2.0.3:*:*:*:*:*:*:*"
          },
          {
            "vulnerable": true,
            "cpe23Uri": "cpe:2.3:a:ibm:data_risk_manager:2.0.4:*:*:*:*:*:*:*"
          },
          {
            "vulnerable": true,
            "cpe23Uri": "cpe:2.3:a:ibm:data_risk_manager:2.0.5:*:*:*:*:*:*:*"
          },
          {
            "vulnerable": true,
            "cpe23Uri": "cpe:2.3:a:ibm:data_risk_manager:2.0.6:*:*:*:*:*:*:*"
          },
          {
            "vulnerable": true,
            "cpe23Uri": "cpe:2.3:a:ibm:data_risk_manager:2.0.1:*:*:*:*:*:*:*"
          }
        ]
      }
    ]
  },
  "impact": {
    "baseMetricV2": {
      "cvssV2": {
        "vectorString": "AV:N/AC:L/Au:S/C:P/I:N/A:N",
        "accessVector": "NETWORK",
        "accessComplexity": "LOW",
        "authentication": "SINGLE",
        "confidentialityImpact": "PARTIAL",
        "integrityImpact": "NONE",
        "availabilityImpact": "NONE",
        "baseScore": 4
      },
      "severity": "MEDIUM",
      "exploitabilityScore": 8,
      "impactScore": 2.9,
      "obtainAllPrivilege": false,
      "obtainUserPrivilege": false,
      "obtainOtherPrivilege": false,
      "userInteractionRequired": false
    }
  },
  "publishedDate": "2020-05-07T20:15Z",
  "lastModifiedDate": "2020-05-08T14:39Z"
}